core/controllers/topics_and_skills_dashboard.py

```diff
@@ -198,33 +198,40 @@ def post(self):
         sort_by = self.payload.get('sort')
         status = self.payload.get('status')
 
-        if (classroom_name is not None and
+        if (
+                classroom_name is not None and
                 not isinstance(classroom_name, python_utils.BASESTRING)):
             raise self.InvalidInputException(
                 'Classroom name should be a string.')
 
-        if (urlsafe_start_cursor is not None and
+        if (
+                urlsafe_start_cursor is not None and
                 not isinstance(urlsafe_start_cursor, python_utils.BASESTRING)):
             raise self.InvalidInputException(
                 'Next Cursor should be a string.')
 
-        if (num_skills_to_fetch is None or
+        if (
+                num_skills_to_fetch is None or
                 not isinstance(num_skills_to_fetch, int)):
             raise self.InvalidInputException(
                 'Number of skills to fetch should be a number.')
 
-        if (keywords is not None and (not isinstance(keywords, list) or not all(
-                [isinstance(keyword, python_utils.BASESTRING)
-                 for keyword in keywords]))):
+        if (keywords is not None and (
```

Comment:
Why we are modifying the first line here? It is OK to break all the last angular bracket.